draft-ietf-tls-tls13.md

```diff
@@ -2301,9 +2303,14 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+ecdsa_sha1
```

Comment:
List structure here is odd: you might want to label this as "Legacy Algorithms" and merge the paragraph that follows the list.

I believe that both follow the rule regarding certificates as well, but this implies that pkcs1 is exempt from the prohibition regarding use in TLS-itself.